src/main/java/ch/njol/skript/patterns/InvalidPatternException.java

```diff
@@ -0,0 +1,33 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.patterns;
+
+import org.eclipse.jdt.annotation.Nullable;
+
+public class InvalidPatternException extends IllegalArgumentException {
```

Comment:
Do you think `MalformedPatternException` should be made public and used in place of this?

Reply:
I think I'd rather see MalformedPatternException be removed and replaced with this class instead, because it'd be better to have the exception in the same location (package) as the pattern compiler.

Reply:
What do you think about renaming this to MalformedPatternException then? Part of me does prefer that name lol - it doesn't really matter though

Reply:
Sure